src/com/nisovin/magicspells/castmodifiers/conditions/VariableCompareCondition.java

```diff
@@ -12,32 +12,67 @@
 
 public class VariableCompareCondition extends Condition {
 
-    String variable;
-    String firstVariable = null;
-    String secondVariable = null;
+    public String variable;
+    public String var = variable;
+    String firstVariable;
+    String secondVariable;
+    double valueDouble;
+    double valueDoubleSecond;
 
     @Override
     public boolean setVar(String var) {
-        try {
+		this.var = var;
+        if(var.contains(":")) {
+            // Find out if its comparing dual values
             String[] split = var.split(":",2);
             firstVariable = split[0]; //The variable that is being checked
             secondVariable = split[1]; //The string that the variable is being checked against
             return true;
-        } catch (ArrayIndexOutOfBoundsException missingColon) {
+        }
+        if(var.contains("<")) {
+            // Find out if its a more than equation
+            String[] split = var.split("<",2);
+            firstVariable = split[0];
+            secondVariable = split[1];
+            return true;
+        }
+        if(var.contains(">")) {
+            // Find out if its a less than equation
+            String[] split = var.split(">",2);
+            firstVariable = split[0];
+            secondVariable = split[1];
+            return true;
+        }
+
             // Someone didn't read the GitHub commit
-            MagicSpells.error("You must use a colon to separate the two variables");
+            MagicSpells.error("You must use either <, >, or : to split the variables");
             return false;
-        }
     }
 
     @Override
     public boolean check(Player player) {
         // Get variable values
         String value = MagicSpells.getVariableManager().getStringValue(firstVariable, player);
         String valueSecond = MagicSpells.getVariableManager().getStringValue(secondVariable, player);
+        double valueDouble;
+        double valueDoubleSecond;
+
+        // Will it require the string to be integers?
+        if(var.contains(">") || var.contains("<")) {
```

Comment:
This calculation should not be redone every time the condition needs to be checked